packages/components/src/components/data-table/_data-table-core.scss

```diff
@@ -264,6 +264,7 @@
     tbody
     tr:not(.#{$prefix}--parent-row):nth-child(even)
     td {
+    // V11: Question for design: This currently maps to $ui-02
     background-color: $data-table-zebra-color;
```

Comment:
@aagonzales the zebra styles currently map to `$ui-02`. Do we have any updated tokens we can use for these?

Reply:
No we don't at the moment. If we keep the light variants we will.

Reply:
Do you know what token we should use for zebra styling then?

Reply:
I've got to think about it a little. We may need to add a new one.

Reply:
Sounds good. I noticed it is also used for the `bx--content` class in the [UI Shell ](https://deploy-preview-8350--carbon-components-react.netlify.app/?path=/story/components-ui-shell--fixed-side-nav) where we have it blend in with the background in the light themes, but higher contrast in dark. Didn't see any current tokens that had the same type of contrast bump between light / dark themes. 

I believe it is also used in [Toggle](https://github.com/carbon-design-system/carbon/pull/8376/files#diff-269c960611bf3ff718a8247dfef83564ca602d1213f00a7c405e9737113a331bR384)